Tests/AuthFoundationTests/TokenTests.swift

```diff
@@ -47,6 +48,22 @@ final class TokenTests: XCTestCase {
         XCTAssertEqual(context, decodedContext)
     }
     
+    func testNilScope() throws {
+        let data = data(for: """
+        {
+           "token_type": "Bearer",
+           "expires_in": 3600,
```

Comment:
Worth noting, some Authorization servers may not return expires_in (notably when you are doing pure OAuth, without OIDC). I don't think it's something we need to account for, but I think the only 2 things that are required from OAuth are access_token/token_type.

Reply:
Yeah, I'm not sure if we should worry about that. There's a lot of code that assumes the `expires_in` exists, for both Swift and Kotlin SDKs.  So I'd hold off on this, until it's necessary.